### DISCUSSION TITLE
Google Chat groups

### DISCUSSION THREAD

**Discussion by SimKay1** [StevenBlack/hosts#2540] [General]
Hi, 

I would like to know if it's possible to blacklist Google chat while allowing Gmail. is it possible? or not possible since it's embedded into Gmail?

thanks

**Comment by welcome** [comment]
Hello!  Thank you for opening your first issue in this repo.  It’s people like you who make these host files better!

**Comment by StevenBlack** [comment]
Hi @SimKay1 thank you for this!

I am converting this to a [discussion](https://github.com/StevenBlack/hosts/discussions) since it's out of scope as an issue.
